src/registry_schemas/schemas/notice_of_withdrawal.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/notice_of_withdrawal",
+    "type": "object",
+    "definitions": {},
+    "title": "Notice of Withdrawal Filing",
+    "required": [
+        "noticeOfWithdrawal"
+    ],
+    "properties": {
+        "noticeOfWithdrawal": {
+            "type": "object",
+            "required": [],
```

Comment:
yeah, I think we need the `filingId` for the withdrew filing. And I think we may need to put the `filingId` as reqruied, otherwise, we won't know which filing should be withdrew